controls/SV-230372.rb

```
@@ -74,14 +74,35 @@
   tag cci: ['CCI-000765']
   tag nist: ['IA-2 (1)']
 
+  pam_auth_files = input('pam_auth_files')
+
   if virtualization.system.eql?('docker')
     impact 0.0
     describe 'Control not applicable within a container' do
       skip 'Control not applicable within a container'
     end
   else
-    describe ini('/etc/sssd/sssd.conf') do
-      its('pam_cert_auth') { should cmp 'True' }
+    if file(input('sssd_conf_path')).exist?
+      describe parse_config_file(input('sssd_conf_path')) do
+        its('pam') { should include('pam_cert_auth' => 'True') }
+      end
+      describe service('sssd') do
+        it { should be_installed }
+        it { should be_enabled }
+        it { should be_running }
+      end
+    else
+      describe "The sssd.conf file was not found at: #{input('sssd_conf_path')}" do
+        skip "The sssd.conf file was not found at: #{input('sssd_conf_path')}"
```

@aaronlippold @wdower Should the control be manual review or fail if the file is not found?

---

I would take out this `skip`. If the user gives a wrong file path, the file resource should handle the error.

Otherwise, to be consistent, we'd have to try to catch wrong paths literally every time we read a file, which we don't do.